phpstan.neon

```diff
@@ -10,6 +10,8 @@ parameters:
 		- PHPStan\Testing\RuleTestCase
 	excludes_analyse:
 		- %currentWorkingDirectory%/test/Fixture/
+	ignoreErrors:
+		- '#Language construct isset\(\) should not be used.#'
```

Comment:
Can't point to specific files yet because of https://github.com/localheinz/phpstan-rules/blob/f8c9435f418b0519b734044e6af1f9f5cf169605/composer.json#L21.